torch/__init__.py

```diff
@@ -193,6 +194,41 @@ def _load_global_deps():
 if False:
     import torch._C as _C
 
+# Check to see if we can load C extensions, and if not provide some guidance
+# on what the problem might be.
+import torch._C as _C_for_compiled_check
+if _C_for_compiled_check.__file__ is None:
+    _cwd_torch_init = os.path.join(os.getcwd(), 'torch', '__init__.py')
+    if __file__ == _cwd_torch_init:
+        raise ImportError(textwrap.dedent(f'''
+            Failed to load PyTorch C extensions:
+                You appear to be importing PyTorch from a clone of the git repo:
+                    {os.getcwd()}
+                If you intended to compile and run PyTorch from within this
+                clone then this means that the compilation has failed.
+                Otherwise, `cd` to a different directory and rerun your code.
+            ''').strip())
+
+    if any(__file__ == os.path.join(i, 'torch', '__init__.py') for i in sys.path):
+        _torch_root_path = os.path.split(os.path.split(__file__)[0])[0]
+        raise ImportError(textwrap.dedent(f'''
+            Failed to load PyTorch C extensions:
+                `{_torch_root_path}`` appears in `sys.path`
```

Comment:
An extra ` here?

Reply:
Done. (My IDE keeps trying to "help" me...)